tests/integ/sagemaker/jumpstart/model/test_jumpstart_model.py

```diff
@@ -262,3 +263,25 @@ def test_jumpstart_model_register(setup):
     response = predictor.predict("hello world!")
 
     assert response is not None
+
+
+@pytest.mark.skipif(
+    True,
+    reason="Only enable if test account is subscribed to the proprietary model",
+)
+def test_proprietary_jumpstart_model(setup):
+
+    model_id = "ai21-jurassic-2-light"
+
+    model = JumpStartModel(
```

Comment:
can you add the JumpStart tags here? Otherwise it won't be marked for deletion and we'll hit resoruce limits?

Reply:
Thanks, added